ParisBNF/abb/BNFabb207.xml

```diff
@@ -0,0 +1,278 @@
+                                            <citedRange unit="page">122</citedRange>
+                                            <citedRange unit="number">207</citedRange>
+                                        </bibl>
+                                        
+                                        <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">206</citedRange>
+                                            <citedRange unit="number">207</citedRange>
+                                        </bibl>
+                                    </listBibl>                                    
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    
+                    <msPart xml:id="p1">
+                        <msIdentifier/>
+                        <msContents>
+                            <summary/>
+                            
+                            <msItem xml:id="p1_i1">                              
+                        <locus from="1r"/>
+                          <title type="complete" ref="LIT1919Mashaf"/>
+                         <textLang mainLang="gez"/>
+                         <note>The same text as in <ref type="mss" corresp="BNFabb69"/>.</note>
+                            </msItem>
+
+                                <msItem xml:id="p1_i2">
+                           <locus from="33r"/>
+                             <title type="complete" ref="LIT2807RepCh91"/>
+                             <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">ሰላም፡ ለኪ፡ ማርያም፡ ድንግል፡ ዘመዐዛ፡ አፉኪ፡ ኮል</incipit>
+                                </msItem>
+
+                                <msItem xml:id="p1_i3">
+                            <locus from="34v"/>
+                                    <title type="complete" ref="LIT7021BaryaAganentGudale"/>
+                              <textLang mainLang="gez"/>
+                              <incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="ellipsis" resp="CR"/>ዛሙስ፡ የአድ፡ ታውሚ፡ አስሆሩ<gap reason="illegible" unit="chars" quantity="1" resp="CR"/>ብለባርያ፡ 
+                                  አገባጅ፡ ዘበላዕለ፡ ምሻቱር፡ ሰሎሞን፡ አርምኃ። <gap reason="ellipsis" resp="CR"/>ወለአጋንንት፡ ለደስክ፡ ወለጉዳሌ፡ ለቡዳ፡ ወለተግባረ፡ ሰብእ፡ ለውግዓት፡ 
+                                  ወለቁርጥማ፡ ለድድቅ፡ ወለጋኔነ፡ ቀትር፡ ለፌራ፡ ወለዳስማ፡ ለምትአት፡ ወለፅፋፍአት፡</incipit>
+                         </msItem>
+
+                                <msItem xml:id="p1_i4">
+                            <locus from="38r"/>
+                                    <title type="incomplete" ref="LIT7022HemamaBedbed">ጸሎት፡ በእንተ፡ ሕማመ፡ ብድብድ፡</title>
+                             <textLang mainLang="gez"/>
+                                    <note>How the Lord spoke to ʾAmru at Mirod, he sent to ʾAmru an angel called Ṭǝqonṭoqyā, 
```

Comment:
Is it better to indicate, that this is a translation of a portion of the text? E.g. 
`'How the Lord spoke to ʾAmru at Mirod, he sent to ʾAmru an angel called Ṭǝqonṭoqyā, (...)'`

Reply:
This is how I styled the renarration of Conti Rossini.